client/app/dbaas/dbaas-metrics/token/metrics-token.less

```diff
@@ -47,6 +48,14 @@
     }
 
   }
+  &__row {
+    .oui-datagrid__row {
+      .oui-input-group_clipboard {
```

Comment:
Must be done in our OVH UI Kit (https://github.com/ovh-ux/ovh-ui-kit/tree/master/packages/oui-datagrid)

Reply:
This style is required only for this table as there are some customized columns in this which breaks the clipboard

Reply:
Could be a case to talk about with @AxelPeter then 😃

Reply:
Margins on this kind of control will be removed in the next releases. 
But why the flex ?

Reply:
The clipboard component has display block and this is actually breaking the table since the other column has variable height and produces an unnecessary scroll bar next to the clipboard column. Hence used display flex on the clipboards so that it fits.

Reply:
I personally prefer adding a small comment `TODO:…` and explain why we have this code and where we have to fix the root cause.